pkg/notes/document/template.go

```diff
@@ -71,12 +71,11 @@ filename | sha512 hash
 {{range .}} {{println "-" .}} {{end}}
 {{end}}
 
-{{- with .Notes -}}
+{{- if .Notes -}}
 ## Changes by Kind
-{{ range .}}
+{{ range .Notes}}
 ### {{.Kind | prettyKind}}
-{{range $note := .NoteEntries }} - {{$note}}{{end}}
-{{- end}}
-{{- end}}
-{{- /* This removes any extra line at the end. */ -}}
+{{range $note := .NoteEntries }}{{println " -" $note}}{{end}}
```

Comment:
This does not work right now, if a note is already prefixed with `- ` then it will result in `- - Note`.

Reply:
@j0n3lson do you see a way to fix this, maybe with a function?

Reply:
Hey 👋, any news on this one? Otherwise I can have a look into it if you want.